check/check.go

```diff
@@ -26,6 +26,40 @@ import (
 
 // State is the state of a control check.
 type State string
+type AuditType string
+
+const TypeAudit = "audit"
+
+type Audit struct {
```

Comment:
Does Audit need to be a struct here? Couldn't it just be a string? (`type Audit string`)

Reply:
I prefer it as a struct to avoid unnecessary casting back to string, also it good example how to create custom Audit.

Reply:
I don't see your argument for being an example of custom Audit - there is nothing particularly special about structs.